contributing/code/standards.rst

```diff
@@ -167,6 +167,31 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-conditions:
+
+Conditions
+----------
+
+Use `Yoda condition`_ when you need to check a variable against a value.
+The main goal with this syntax is to avoid an accidental assignment inside the
+condition statements:
+
```

Comment:
What if we reword this paragraph to make it much shorter?

```
Use `Yoda conditions`_ when checking a variable against a value to avoid an
accidental assignment inside the condition statements:
```